src/client/components/forms/userCollection.js

```diff
@@ -0,0 +1,306 @@
+		}
+		const newCollaborators = this.state.collection.collaborators;
+		newCollaborators[index] = newCollaborator;
+		this.setState(prevState => ({
+			collection: {
+				...prevState.collection,
+				collaborators: newCollaborators
+			},
+			errorText: prevState.errorText
+		}));
+	}
+
+	render() {
+		if (this.state.collection.collaborators.length === 0) {
+			this.handleAddCollaborator();
+		}
+
+		const privacyOptions = ['Private', 'Public'].map((option) => ({
+			name: option
+		}));
+		const entityTypeOptions = ['Author', 'Work', 'Edition', 'Edition-Group', 'Publisher'].map((entity) => ({
+			name: entity
+		}));
+		const initialName = this.state.collection.name;
+		const initialDescription = this.state.collection.description;
+		const initialPrivacy = this.state.collection.public ? 'Public' : 'Private';
+		const initialType = this.state.collection.entityType;
+		const {errorText} = this.state;
+		const errorAlertClass =
+			classNames('text-center', 'margin-top-1', {hidden: !errorText});
+		const submitLabel = this.props.collection.name ? 'UPDATE' : 'CREATE';
+
+		/* eslint-disable react/jsx-no-bind */
+		return (
+			<Grid>
+				<h1>Create a collection</h1>
+				<Row>
+					<Col md={12}>
+						<p className="lead">Create Your Collection</p>
+					</Col>
+				</Row>
+				<Row>
+					<Col
+						id="collectionForm"
+						md={8}
+						mdOffset={2}
+					>
+						<form
+							className="form-horizontal"
+							onSubmit={this.handleSubmit}
+						>
+							<CustomInput
+								defaultValue={initialName}
+								label="Name"
+								ref={(ref) => this.name = ref}
+								type="text"
+							/>
+							<CustomInput
+								defaultValue={initialDescription}
+								label="Description"
+								ref={(ref) => this.description = ref}
+								type="textarea"
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialType}
+								disabled={!this.props.canEditType}
+								idAttribute="name"
+								label="Entity Type"
+								labelAttribute="name"
+								options={entityTypeOptions}
+								placeholder="Select Entity Type"
+								ref={(ref) => this.entityType = ref}
+							/>
+							<SelectWrapper
+								base={ReactSelect}
+								defaultValue={initialPrivacy}
+								idAttribute="name"
+								label="Privacy"
+								labelAttribute="name"
+								options={privacyOptions}
+								placeholder="Select Privacy"
+								ref={(ref) => this.privacy = ref}
+							/>
+							{
+								this.state.collection.collaborators.map((collaborator, index) => (
+									<div key={collaborator.id}>
+										<EntitySearchFieldOption
+											instanceId="collaboratorSearchField"
+											label="Select Collaborator"
+											name="editor"
+											type="editor"
+											value={collaborator}
+											onChange={(newCollaborator) => this.handleChangeCollaborator(newCollaborator, index)}
+										/>
+										<Button
+											bsSize="small"
+											bsStyle="primary"
```

Comment:
```suggestion
											bsStyle="danger"
```

You could also add a trash bin icon like in some other pages